dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/validate-csv-data/cells/index.test.ts

```diff
@@ -0,0 +1,59 @@
+import { UTILISATION_REPORT_HEADERS } from '@ukef/dtfs2-common';
+import validateCells from '.';
+import {
+  generateUkefFacilityIdError,
+  generateBaseCurrencyError,
+  generateFacilityUtilisationError,
+  generatePaymentCurrencyError,
+  generatePaymentExchangeRateError,
+} from '../..';
+
+jest.mock('../..', () => ({
+  generateUkefFacilityIdError: jest.fn(),
+  generateBaseCurrencyError: jest.fn(),
+  generateFacilityUtilisationError: jest.fn(),
+  generatePaymentCurrencyError: jest.fn(),
+  generatePaymentExchangeRateError: jest.fn(),
+  generateTotalFeesAccruedCurrencyError: jest.fn(),
+  generateTotalFeesAccruedExchangeRateError: jest.fn(),
+}));
```

Comment:
Baffles me why we need this amount of mocks.

For this PR, are these existing and have just been moved?

Reply:
these are existing and just been moved